ModuleChangelog.md

```diff
@@ -1,3 +1,15 @@
+## 2.7.6
+
+Release Date: August 21, 2024
+
+#### RELEASE NOTES
+
+This fixes an issue with disabled Migrate Button
```

Comment:
```suggestion
This release addresses a UI issue with the ADMU where the "Migrate Profile" button could not be selected even though all the required fields were satisfied.
```

Reply:
The issue was that Test-Button needed an Org ID for the if statement logic to enable the migrate button. Before this change, Test-Button gets called before initializing the OrgID when API key is inputted. Fix was to move the Test-Button after OrgId is initialized.

Reply:
Thank you!